packages/components/src/components/input-radio/controller.ts

```diff
@@ -124,21 +98,18 @@ export class InputRadioController extends InputCheckboxRadioController implement
 		value = Array.isArray(value) ? (value[0] as StencilUnknown) : value;
 		setState(this.component, '_value', value, {
 			afterPatch: this.afterPatchOptions,
-			beforePatch: this.beforePatchOptions,
 		});
 	}
 
 	public componentWillLoad(onChange?: (event: Event) => void): void {
 		super.componentWillLoad();
 
-		this.onStateChange = () => {
-			if (typeof onChange === 'function') {
-				const timeout = setTimeout(() => {
-					clearTimeout(timeout);
-					onChange(STATE_CHANGE_EVENT);
-				});
-			}
-		};
+		if (typeof onChange === 'function') {
```

Comment:
Das sieht für mich nicht richtig aus: Vorher wurde onChange aufgerufen, wenn die Optionen sich geändert haben, jetzt direkt on load?

Ich vermute, wir können dieses Code-Fragment einfach entfernen, weil onChange bereits in der Komponente selbst aufgerufen wird. Das sollte aber (in der Sample App) getestet werden.